usaspending_api/search/helpers/spending_by_category_helpers.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from typing import Tuple
+
+from usaspending_api.recipient.models import StateData
+from usaspending_api.references.models import Agency, Cfda, PSC, RefCountryCode, NAICS
+
+logger = logging.getLogger(__name__)
+
+
+def fetch_agency_tier_id_by_agency(agency_name: str, is_subtier: bool = False) -> int:
```

Comment:
Any function which includes
```python
return None
```

needs wrap the return type with `Optional[]` 
https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type